e2e/utils.go

```diff
@@ -542,6 +575,22 @@ func validatePVCClone(sourcePvcPath, clonePvcPath, clonePvcAppPath string, f *fr
 		go func(w *sync.WaitGroup, n int, p v1.PersistentVolumeClaim, a v1.Pod) {
 			name := fmt.Sprintf("%s%d", f.UniqueName, n)
 			wgErrs[n] = createPVCAndApp(name, f, &p, &a, deployTimeout)
+			pvcClone, err = f.ClientSet.CoreV1().PersistentVolumeClaims(pvc.Namespace).Get(context.TODO(), pvcClone.Name, metav1.GetOptions{})
+			if err != nil {
+				e2elog.Failf("failed to get pvc %v", err)
```

Comment:
@yati1998 Recent test is failing here, saying that PVC is not found.

I think instead of pvcClone, you should use pvc.Name only. Can you please try with that? 

Dec 10 13:17:53.389: FAIL: failed to get pvc persistentvolumeclaims "rbd-pvc-clone" not found

/go/src/github.com/ceph/ceph-csi/e2e/**utils.go:580** +0x378
created by github.com/ceph/ceph-csi/e2e.validatePVCClone
/go/src/github.com/ceph/ceph-csi/e2e/utils.go:575 +0xbbb
make: *** [Makefile:164: run-e2e] Error 2
script returned exit code 2

Reply:
Or maye be this load is not required if we are using pvc only in place of pvcClone.
Lets just go back to the previous code (whetever you had previously) for this segment and try once.

Reply:
@agarwal-mudit yes I was going through the same, I have noticed that I have used pvc.Namespace but elsewhere it is pvcClone, Hence I have tried changing that part. I guess this will work, If not will try the way you mentioned.